OrbitSshQt/include/OrbitSshQt/SftpCopyToRemoteOperation.h

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef ORBIT_SSH_QT_SFTP_OPERATION_H_
-#define ORBIT_SSH_QT_SFTP_OPERATION_H_
+#ifndef ORBIT_SSH_QT_SFTP_COPY_TO_REMOTE_OPERATION_H_
```

Comment:
What about moving these changes into the first commit?

Reply:
done.